theano/sandbox/mkl/__init__.py

```diff
@@ -0,0 +1,254 @@
+from __future__ import absolute_import, print_function, division
+import os
+import errno
+import sys
+import logging
+import textwrap
+import stat
+import shutil
+import theano
```

Comment:
nitpik, add an empty line before the import of Theano to separate the different import group. (we aren't always consistent in Theano, but we try to go in that direction, at one point, we would probably do it in all files)